### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by InWamos**
### Pull Request Overview

This pull request adds/changes/fixes...

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs...

### Build

- [x] Ran `npm build`.

### Author

Signed-off-by: Mykyta Pidgornyi <mykyta.pidgornyi@gmail.com>

### CODE REVIEW

File: docs/project/2024/mykyta.pidgornyi/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+| Library                                                                     | Description               | Usage                                           |
+| --------------------------------------------------------------------------- | ------------------------- | ----------------------------------------------- |
+| [st7789](https://github.com/almindor/st7789)                                | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [GPS](https://wiki.dfrobot.com/SKU_TEL0157_Gravity_GNSS_Positioning_Module) | Drivers for GPS           | Used as a GPS driver                            |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+https://www.youtube.com/watch?v=gBofy7MMdIY
```
Comment (alexandruradovici) [inline docs/project/2024/mykyta.pidgornyi/index.md:97]: Add a title to the link using the markdown syntax.

File: docs/project/2024/mykyta.pidgornyi/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+| [TEL0157](https://www.digikey.ro/en/products/detail/dfrobot/TEL0157/18069252?s=N4IgTCBcDaICoFEAyAGAjAVgOwgLoF8g)                               | GPS Module                                   | 83,24 RON  |
+| [PICO EXPLORER BASE](https://www.digikey.ro/en/products/detail/pimoroni-ltd/PIM550/13981737?s=N4IgTCBcDaIAoEkCyBWFAGEBdAvkA)                   | Base for Microcontroller                     | 133,92 RON |
+| [Wires MALE - FEMALE](https://www.digikey.ro/ro/products/detail/sparkfun-electronics/PRT-12794/5993859?s=N4IgTCBcDaIIwFYBsAOAtIuc0DkAiIAugL5A) | Connector from a component to the breadboard | 9,77 RON   |
+| [Wires MALE - MALE](https://www.digikey.ro/ro/products/detail/adafruit-industries-llc/1957/6827090?s=N4IgTCBcDaIIwFYwA4C0cCcA2A7KgcgCIgC6AvkA) | Connector from a component to the breadboard | 9,07 RON   |
+
+
+## Software
+
+| Library                                                                     | Description               | Usage                                           |
+| --------------------------------------------------------------------------- | ------------------------- | ----------------------------------------------- |
```
Comment (alexandruradovici) [inline docs/project/2024/mykyta.pidgornyi/index.md:89]:
```suggestion
|-|-|-|
```

File: docs/project/2024/mykyta.pidgornyi/index.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device                                                                                                                                         | Usage                                        | Price      |
+| ---------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- | ---------- |
```
Comment (alexandruradovici) [inline docs/project/2024/mykyta.pidgornyi/index.md:78]:
```suggestion
|-|-|-|
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/mykyta.pidgornyi/Pasted image 20240504182423.png
Comment (alexandruradovici) [inline docs/project/2024/mykyta.pidgornyi/Pasted image 20240504182423.png:1]: Do not use spaces in file names.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/mykyta.pidgornyi/index.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+| Library                                                                     | Description               | Usage                                           |
+|-|-|-|
+| [st7789](https://github.com/almindor/st7789)                                | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [GPS](https://wiki.dfrobot.com/SKU_TEL0157_Gravity_GNSS_Positioning_Module) | Drivers for GPS           | Used as a GPS driver                            |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+[https://www.youtube.com/watch?v=gBofy7MMdIY](Thermometer and Digital Clock)
```
Comment (alexandruradovici) [inline docs/project/2024/mykyta.pidgornyi/index.md:97]: Please take a look at the markdown link syntax.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by TeodoraMiu — APPROVED** [review]

**Pull request merged**